jina/drivers/cache.py

```diff
@@ -42,7 +41,7 @@ def _apply_all(self, docs: 'DocumentSet', *args, **kwargs) -> None:
                     self.on_hit(d, result)
 
     def on_miss(self, doc: 'Document', data) -> None:
-        """Function to call when doc is missing, the default behavior is add to cache when miss
+        """Function to call when document is missing, the default behavior is add to cache when miss.
```

Comment:
'the default behavior is add to cache when miss. '
does it mean 'the default behavior is to add the document to cache when miss. '?